src/System.Numerics.Vectors/tests/GenericVectorTests.cs

```diff
@@ -444,45 +444,6 @@ static GenericVectorTests()
         #endregion
 
         #region System.Object Overloads
-        [Fact]
-        public void GetHashCodeByte() { TestGetHashCode<Byte>(); }
-        [Fact]
-        public void GetHashCodeSByte() { TestGetHashCode<SByte>(); }
-        [Fact]
-        public void GetHashCodeUInt16() { TestGetHashCode<UInt16>(); }
-        [Fact]
-        public void GetHashCodeInt16() { TestGetHashCode<Int16>(); }
-        [Fact]
-        public void GetHashCodeUInt32() { TestGetHashCode<UInt32>(); }
-        [Fact]
-        public void GetHashCodeInt32() { TestGetHashCode<Int32>(); }
-        [Fact]
-        public void GetHashCodeUInt64() { TestGetHashCode<UInt64>(); }
-        [Fact]
-        public void GetHashCodeInt64() { TestGetHashCode<Int64>(); }
-        [Fact]
-        public void GetHashCodeSingle() { TestGetHashCode<Single>(); }
-        [Fact]
-        public void GetHashCodeDouble() { TestGetHashCode<Double>(); }
-        private void TestGetHashCode<T>() where T : struct
```

Comment:
We are removing these tests here. Should we instead just correct them based on the using the new HashCode.Combine or is that unnecessary?

Reply:
Actually, I might be able to get these to work. The problem is the seed randomization that HashCode performs.